corehq/apps/reports/generic.py

```diff
@@ -857,11 +858,20 @@ def export_table(self):
         3. str(cell)
         """
         headers = self.headers
+        # Strip HTML tags using regex. Regex is much faster than using an HTML
+        # parser, but will strip "<2 && 3>" from a value like "1<2 && 3>2". A
+        # parser will treat each cell like an HTML document, which might be
+        # overkill, but if using regex breaks values then we should use a
+        # parser instead, and take the knock. Assuming we won't have values
+        # with angle brackets, using regex for now.
+        tag_pattern = re.compile('<[^>]*?>')
```

Comment:
can you extract this into a function called `strip_simple_html` or something equivalent and then write a few tests for it? it would be good to understand what we think we support.